### ISSUE TITLE
cdecker's GPG key is expired and the automatic tag verification for v22.11 fails

### ISSUE THREAD

**Issue by 1ma** [ElementsProject/lightning#5769]
### Issue and Steps to Reproduce

```bash
$ gpg --recv-key A26D6D9FE088ED58
gpg: key A26D6D9FE088ED58: public key "Christian Decker <decker.christian@gmail.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1

$ git clone https://github.com/ElementsProject/lightning
$ cd lightning
$ git verify-tag v22.11
gpg: Signature made Wed Nov 30 19:45:00 2022 UTC
gpg:                using RSA key B7C4BE81184FC203D52C35C51416D83DC4F0E86D
gpg: Good signature from "Christian Decker <decker.christian@gmail.com>" [unknown]
gpg:                 aka "Christian Decker <decker@blockstream.com>" [unknown]
gpg: Note: This key has expired!
Primary key fingerprint: B731 AAC5 21B0 1385 9313  F674 A26D 6D9F E088 ED58
     Subkey fingerprint: B7C4 BE81 184F C203 D52C  35C5 1416 D83D C4F0 E86D

$ echo $?
1
```

**Comment by cdecker** [comment]
Should be addressed by #5771 :+1:

**Comment by 1ma** [comment]
The changes haven't propagated through the keyservers yet, but I confirm that the updated `contrib/keys/cdecker.txt` in this PR works fine.
